File: examples/python/slate_example.py
import os
import uuid
import requests

base_url = os.environ["SLATE_BASE_URL"].rstrip("/")
api_key = os.environ["SLATE_API_KEY"]

response = requests.post(
    f"{base_url}/api/v1/ai/gateway",
    headers={"Authorization": f"Bearer {api_key}"},
    json={
        "employeeExternalId": "employee-427",
        "clientExternalId": "client-acme",
        "projectExternalId": "project-seo-001",
        "workflowExternalId": "client-update",
        "taskType": "client_update",
        "sourceAppRequestId": str(uuid.uuid4()),
        "model": "gpt-4o-mini",
        "input": "Draft a brief client update.",
    },
    timeout=35,
)
response.raise_for_status()
print(response.json()["response"])

File: examples/typescript/slate-example.ts
import { SlateClient } from "@slate-ai/sdk";

const slate = new SlateClient({
  apiKey: process.env.SLATE_API_KEY!,
  baseUrl: process.env.SLATE_BASE_URL!,
});

const context = await slate.getContext();
const employee = context.employees[0];

if (!employee) throw new Error("Sync at least one employee first.");

const result = await slate.run({
  employeeExternalId: employee.externalId,
  sourceAppRequestId: crypto.randomUUID(),
  input: "Draft a concise internal status update.",
});

console.log(result.response);
